cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectS2IAssemblyOperator.java

```diff
@@ -80,6 +80,7 @@ public void createOrUpdate(Reconciliation reconciliation, ConfigMap assemblyCm,
 
             deploymentConfigOperations.scaleDown(namespace, connect.getName(), connect.getReplicas())
                     .compose(scale -> serviceOperations.reconcile(namespace, connect.getName(), connect.generateService()))
+                    .compose(i -> configMapOperations.reconcile(namespace, connect.getMetricsConfigName(), connect.generateMetricsConfigMap()))
```

Comment:
This is good. But I think you have to also update the methods `delete` and `getResources`. The config map is now part of the deployment and should be deleted / listed.

Reply:
Done